README.md

```diff
@@ -120,6 +90,40 @@ git merge <name-of-branch>
 
 You may need to fix merge conflicts if new updates to files were affected by your changes.
 
+## 👹 Hasura Migrations
+
+Important files related to hasura are the `config.yaml` file at the root of the site and the `migrations` folder. The `config.yaml` lists an endpoint to interact with when you run commands from the console. There are 3 Hasura apps deployed:
+
+- https://recipe-palette-dev.herokuapp.com/
+- https://recipe-palette-stg.herokuapp.com/
+- https://recipe-palette.herokuapp.com/
+
+These endpoints all correspond to an app on Heroku that is running an identical instance of the Hasura `graphql-engine`.
+
+By keeping individual apps we can update the schema, or play with data in the database while updating GraphQL queries without breaking production.
+
+In order to make changes to the database don't use the link at https://recipe-palette.herokuapp.com/console, this will lead to inconsistent versions of the schema. We can turn this off with an `HASURA_GRAPHQL_ENABLE_CONSOLE=false` environment variable. It's on so we can still use GraphiQL and easily view the schema. We can change it if need be.
```

Comment:
Ok. So no online database changes? The right way is to call it from the hasura console. I like that it will pull up a GUI to go about the changes.

Reply:
Right! We could use that link but it would start getting confusing. So you use the `hasura console` command and it opens a window to do the same thing.